pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -47,17 +47,23 @@ buildDotnetModule rec {
   '';
 
   postPatch = ''
+    # We still need this for the tests, even though we build with NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR
+    # See https://github.com/Nexus-Mods/NexusMods.App/issues/1836
     ln --force --symbolic "${lib.getExe _7zz}" src/ArchiveManagement/NexusMods.FileExtractor/runtimes/linux-x64/native/7zz
```

Comment:
Is there a (sane) way to symlink 7zip only for the tests without affecting the main package?